### PR TITLE
Migrate from travis to github actions

### LINKED ISSUE #296: Switch to github action
Travis is way too slow nowadays

### PR DESCRIPTION
**Pull request by RomainGoussault**
Close #296

### CODE REVIEW

File: .github/workflows/main_ci.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install libsndfile
+      run: |
+        sudo apt-get install libsndfile1
```
Comment (natct10) [inline .github/workflows/main_ci.yml:21]: Is this related to librosa lib?

Comment (RomainGoussault) [reply .github/workflows/main_ci.yml:21]: Yes i added it because of a missing sound lib

**Review by arthurPignet — APPROVED** [review]
